recipe/meta.yaml

```diff
@@ -38,10 +38,9 @@ requirements:
     - python  # [win]
     - meson   # [win]
     - ninja   # [win]
-
   host:
     - freetype
-    - libxml2
+    - libxml2 {{ libxml2 }}
```

Comment:
please replace jinja2 libxml2 by the version directly, and remove the entry from recipe's cbc.
additionally make sure an abs.yaml file is turning off overdepending check

Reply:
I replaced  jinja2 libxml2 by the version directly. abs.yaml already exists with disabling overdepending checks

Reply:
Thanks. I forgot about that. Added to my bookmarks.